tracing/src/span.rs

```diff
@@ -720,10 +720,41 @@ impl Span {
     ///
     /// If this span is disabled, or the resulting follows-from relationship
     /// would be invalid, this function will do nothing.
-    pub fn follows_from(&self, from: impl for<'a> Into<Option<&'a Id>>) -> &Self {
+    ///
+    /// # Example
+    ///
+    /// Setting a `follows_from` relationship through an `Option<Id>`:
```

Comment:
I _think_ it should also be possible to pass in an `Id` (or even an `&Id`?), can we check if that's the case?

Reply:
It _just_ works with `Id`, not `&Id`. I've added tests an a note that cloning `Id` is cheap.